tsconfig.base.json

```diff
@@ -44,6 +44,7 @@
       "@ngrx/schematics/schematics-core": [
         "./modules/schematics/schematics-core"
       ],
+      "@ngrx/schematics-core/*": ["./modules/schematics-core/*"],
```

Comment:
👍
Do we still need the other `schematics-core` paths? e.g. `@ngrx/component/schematics-core`
I think it would be better to just use one path for it, otherwise it might be confusing.

Reply:
Yea, we still need those as `@ngrx/schematics-core` isn't a published package